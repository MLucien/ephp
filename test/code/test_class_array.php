<?php

class Simple {
    public $data = array();

    public function __construct() {
        $this->data = array(
            "hi" => "world"
        );
    }
}

$simple = new Simple();
var_dump($simple->data);
var_dump($simple->data['hi']);

