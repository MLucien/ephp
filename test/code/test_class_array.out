array(1) {
  ["hi"]=>
  string(5) "world"
}
string(5) "world"
